crates/metrics/src/lib.rs

```diff
@@ -0,0 +1,110 @@
+use jemalloc_ctl::{epoch, epoch_mib, stats, thread};
```

Comment:
Setting the global allocator seems like it should be in the `bindings` crate.

As it is, it adds a jemalloc dependency on the `derive` crate.